README.md

```diff
@@ -116,7 +116,7 @@ const Form = withNextInputAutoFocusForm(View);
 We can also create a validation schema, with `yup`. It's of course possible to use other validation possibilities provided by Formik, but `yup` makes validation and error messaging painless.
 
 ```javascript
-import Yup from 'yup';
+import * as Yup from 'yup';
```

Comment:
Hey @pierpo, sorry I don't understand why you need this. This example works well for me on node 8 or 10.
Is this needed only with TypeScript? Ping @VincentLanglet